packages/controllers/src/snaps/SnapController.ts

```diff
@@ -911,13 +915,17 @@ export class SnapController extends BaseController<
    *
    * @param snapId - The snap id to check.
    * @param version - The version of the snap to check.
+   * @param sourceShaSum - The shasum of the snap's source code.
    * @returns Whether the version of the snap is blocked or not.
    */
   async isBlocked(
```

Comment:
Perhaps this should be an options bag now with named parameters instead of positional?

Reply:
Updated now with type parameter `SnapInfo`. It should be the safest. Let me know if you would still like it to be different with an example.